src/main/java/ru/spbstu/dto/AnnotatedVariantList.java

```diff
@@ -0,0 +1,19 @@
+package ru.spbstu.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import ru.spbstu.model.Variant;
+
+import java.util.List;
+
+public class AnnotatedVariantList {
+    @JsonProperty("variantAnnotations")
+    public List<Variant> variants;
```

Comment:
Can we make this field `final` ?

Reply:
Unfortunately, I cannot make this field final because Jackson requires empty constructor for JSON-deserialization that could not be created to class with final fields.
So I used @Value annotation to make object of VariantListDto immutable.